errors.go

```diff
@@ -89,6 +89,10 @@ const (
 	ErrOffsetsLoadInProgress           KError = 14
 	ErrConsumerCoordinatorNotAvailable KError = 15
 	ErrNotCoordinatorForConsumer       KError = 16
+	ErrInvalidTopic                    KError = 17
+	ErrMessageSetSizeTooLarge          KError = 18
+	ErrNotEnoughReplicas               KError = 19
+	ErrNotEnoughReplicasAfterAppend    KError = 20
```

Comment:
These two both occur if you configure `min.insync.replicas` on the topic (or globally) to be >1 and not enough replicas are remaining in-sync. They should be retried in the producer.